certbot/setup.py

```diff
@@ -40,16 +40,16 @@ def read_file(filename, encoding='utf8'):
     # saying so here causes a runtime error against our temporary fork of 0.9.3
     # in which we added 2.6 support (see #2243), so we relax the requirement.
     'ConfigArgParse>=0.9.3',
-    'configobj',
-    'cryptography>=1.2.3',  # load_pem_x509_certificate
+    'configobj>=5.0.6',
```

Comment:
The highest version of configobj for CentOS 7 that I can find (in EPEL) is 4.7.2.

Reply:
I only found a Python 2 package for configobj on CentOS 7 and configobj<5.0 doesn't seem to work on Python 3.

In the case where the package isn't available for Python 3 yet on CentOS 7 + EPEL, I took the approach I described in my initial post:

> For the ~~latter~~ CentOS 7 + EPEL 7, not all packages are available in Python 3 yet (which is something Certbot's EPEL package maintainers are working on) and in that case I didn't worry about the system because I think they can/will package the newest version available. If they end up hitting any issues here when trying to package Certbot on Python 3, we can always work with them to fix it.